Kickstarter-iOS/Views/Cells/MostPopularSearchProjectCell.swift

```diff
@@ -77,15 +86,28 @@ internal final class MostPopularSearchProjectCell: UITableViewCell, ValueCell {
 
     _ = self.separateView
       |> separatorStyle
+
+    _ = self.progressStaticView
+      |> UIView.lens.backgroundColor .~ .black
+      |> UIView.lens.alpha .~ 0.15
   }
 
   internal override func bindViewModel() {
     super.bindViewModel()
 
-    self.projectNameLabel.rac.text = self.viewModel.outputs.projectNameLabelText
-    self.fundingTitleLabel.rac.attributedText = self.viewModel.outputs.fundingLargeLabelText
-    self.deadlineSubtitleLabel.rac.text = self.viewModel.outputs.deadlineSubtitleLabelText
-    self.deadlineTitleLabel.rac.text = self.viewModel.outputs.deadlineTitleLabelText
+    self.metadataBackgroundView.rac.backgroundColor = self.viewModel.outputs.progressBarColor
+    self.metadataLabel.rac.text = self.viewModel.outputs.metadataText
+    self.percentFundedLabel.rac.attributedText = self.viewModel.outputs.percentFundedText
+    self.progressBarView.rac.backgroundColor = self.viewModel.outputs.progressBarColor
     self.projectImageView.rac.imageUrl = self.viewModel.outputs.projectImageUrlFull
+    self.projectNameLabel.rac.text = self.viewModel.outputs.projectName
+
+    self.viewModel.outputs.progress
+      .observeForUI()
+      .observeValues { [weak element = progressBarView] progress in
+        let anchorX = progress == 0 ? 0 : 0.5 / progress
```

Comment:
Ok cool :) actually even a tuple value as the output could also work,  no need for separate outputs.